src/script_lua.c

```diff
@@ -431,43 +434,63 @@ static void redisProtocolToLuaType_Double(void *ctx, double d, const char *proto
  * with a single "err" field set to the error string. Note that this
  * table is never a valid reply by proper commands, since the returned
  * tables are otherwise always indexed by integers, never by strings. */
-void luaPushError(lua_State *lua, char *error) {
+void luaPushErrorBuff(lua_State *lua, sds err_buffer) {
```

Comment:
let's mention that the function takes ownership of `err_buffer`

Reply:
Fixed